app/queries.js

```diff
@@ -338,6 +365,39 @@ var RouteMarkerPopupFragments = {
   `,
 }
 
+class FavouriteRouteRowRoute extends Relay.Route {
```

Comment:
You could do the same improvements as in 4c1812fc9b1206ec63f981300ae4058ebf4a3142 for this route also